freva-client/src/freva_client/query.py

```diff
@@ -733,17 +712,111 @@ def _facet_search(
             constraints = data["primary_facets"]
         return {f: v for f, v in data["facets"].items() if f in constraints}
 
-    def _get(
-        self, url: str, **kwargs: Any
-    ) -> Optional[requests.models.Response]:
+    def add_user_data(
+        self, username: str, paths: List[str], facets: Dict[str, str]
+    ) -> None:
+        """Add user data to the databrowser.
```

Comment:
In the cli you used the term "into", "to" is also correct. But let us stick to one, though.